### DISCUSSION TITLE
Can't login with admin credentials

### DISCUSSION THREAD

**Discussion by DDoerner** [plausible/analytics#1024] [Self-Hosted Support] [answered]
Hello people,

I've followed the [self-hosted guide](https://plausible.io/docs/self-hosting) to host the platform on an AWS EC2 instance. Everything seemed to go fine, as shown on the end of the logs:

> Creation of Db successful!
> Loading plausible..
> Starting dependencies..
> Starting repos..
> Running migrations for Elixir.Plausible.Repo
> Running migrations for Elixir.Plausible.ClickhouseRepo
> Migrations successful!
> Loading plausible..
> Starting dependencies..
> Starting repos..
> Admin user created successful!

I'm now trying to login with the credentials set in the conf file, but always get a 403 Forbidden after sending the login form.

I've manually set email_verified, I've manually deleted the admin user and ran the compose script again to recreate it and I've tried multiple times to remove all containers and recreate them as well. I also can't register a new user using the login interface, as that returns a 403 Forbidden after sending the form as well.

Any ideas what might be happening? The plausible_1 and plausible_db_1 container logs don't show anything regarding this.

**Comment by DDoerner** [comment, marked as answer]
Ok, I've found the issue and it wasn't with Plausible. 
For anyone else experiencing the same when running Plausible behind a reverse proxy (we had a Cloudfront instance to enforce HTTPS), make sure that the proxy does not remove HTTP Headers for the login flow. 
We updated the distribution to pass on all headers and now it's working fine.

**Reply by metmarkosaric** [reply]
thanks for the update @DDoerner! enjoy Plausible!
